owsi-core/owsi-core-examples/wicket-showcase/wicket-showcase-webapp/src/main/java/fr/openwide/core/showcase/web/application/renderer/service/RendererServiceImpl.java

```diff
@@ -0,0 +1,13 @@
+package fr.openwide.core.showcase.web.application.renderer.service;
+
+import fr.openwide.core.showcase.web.application.ShowcaseApplication;
+import fr.openwide.core.wicket.more.rendering.service.AbstractRendererServiceImpl;
+
+public class RendererServiceImpl extends AbstractRendererServiceImpl {
+	
+	@Override
+	protected String getApplicationName() {
```

Comment:
Maybe we should consider moving this data to some kind of global variable? Like an environment property that would be set when starting Spring? This would exempt us from adding this kind of method to every service that requires a Wicket context in order to run.
This would make sense, in my opinion, since we rarely (never?) have two wicket applications. Even if we did, I doubt the context thing would work out-of-the-box : we'd probably have to either arbitrarily choose one application (which would be the same as here with an environment variable), or add some kind of mechanism to dynamically choose the proper one (which would require some data to feed in the choice process, and we currently don't have any in `getApplicationName()`).